bindata/assets/jobset-controller/config.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jobset-manager-config
```

Comment:
nit: Newline

I think, we need to disable internal cert manager in here and explicitly enable leader election?